realtime-sanitizer/llvm-project
# Add a few more lit tests

These tests:
1. Test `continue` mode
2. Ensure that each of the attributes work with/without linking the sanitizer library
3. Ensure nosanitize works outside the [[realtime]] block


Just some basic sanity checks to make sure we don't regress! Please double check that we like the test file names
